### ISSUE TITLE
Zxing Library is in maintenance mode, should we change our library? 

### ISSUE THREAD

**Issue by jmbrunskill** [msupply-foundation/open-msupply#5391]
## The suggestion

Investigate changing from ZXing to another library (would require a capcitor plugin too I think?)
https://github.com/zxing/zxing

### Why should we invest time in this?

Support for future platforms and formats might no be available in this library

### Are there any risks associated with this change?

Changing to another library could be less reliable, or cause issues with barcode scanning for our customers.
Care should be taken that we don't make the problem worse by choosing another library that's not well maintained, buggy, or out of date.

### How much effort is required?

Could be significant, if there's not an easily found replacement.

### Agreed Solution

Sounds like it's in maintenance mode, so not completely dead yet. I think we can back burner this...

**Comment by mark-prins** [comment]
yes, absolutely. putting as low priority mainly due to current workload

**Comment by jmbrunskill** [comment]
Now usesing MLKit
